PhotoEnhancer/PhotoEnhancer/Filters/ContrastParameters.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class ContrastParameters: IParameters
+    {
+        [ParameterInfo(Name ="Коэффициент", MinValue =0, MaxValue =10, 
+            DefailtValue =1, Increment =0.05)]
+
+        public double Coefficient { get; set; }
```

Comment:
Атрибуты не принято отделять пустой строкой от объекта к которому они относятся.
```suggestion
        [ParameterInfo(Name ="Коэффициент", MinValue =0, MaxValue =10, 
            DefailtValue =1, Increment =0.05)]
        public double Coefficient { get; set; }
```